spec/system/planning_applications/assessing/adding_informatives_spec.rb

```diff
@@ -173,7 +173,7 @@
       let!(:informative_two) { create(:informative, informative_set:, title: "Title 2", text: "Text 2", position: 2) }
       let!(:informative_three) { create(:informative, informative_set:, title: "Title 3", text: "Text 3", position: 3) }
 
-      it "I can drag and drop to sort the informatives", :capybara do
+      it "I can drag and drop to sort the informatives", :capybara, pending: "flaky" do
```

Comment:
```suggestion
      it "I can drag and drop to sort the informatives", :capybara, skip: "flaky" do
```

`pending` will fail if the test passes (it assumes when it passes you've implemented the thing so need to remove the `pending`). We need `skip` for tests that only fail some of the time.

Reply:
Woops, thanks